build/xwalk.gni

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains Crosswalk-feature-related build flags. Users should
+# import it into their build folder by run:
+#   echo 'import("//xwalk/build/xwalk.gni")' > out/your_folder/args.gn'
+# Then run:
+#   gn gen out/your_folder
+
+# Set it false when building content_shell.
+declare_args() {
+  is_build_crosswalk = true
+}
+
+if (is_build_crosswalk) {
+  target_os = "linux"
```

Comment:
`is_debug` should be same as upstream.
My understanding:  User should set `target_os/target_cpu/is_debug` by` "gn args"`.  Now we use this `"echo import(xwalk.gni)"` approach, User just need to edit this xwalk.gni to modify the values of this 3 lines. 
Another reason is, some flags have different values for different platforms.  for example:
```
 ['OS=="android"', {
     'release_unwind_tables%': 0,
}]
```
By add defines of `target_os`,  I can later write as: 
`if(target_os == Android) { release_unwind_tables = false }`